Source/NSOutlineView.m

```diff
@@ -2202,6 +2208,46 @@ - (NSCell *) preparedCellAtColumn: (NSInteger)columnIndex row: (NSInteger)rowInd
   return cell;
 }
 
+- (NSView *) viewAtColumn: (NSInteger)column row: (NSInteger)row makeIfNecessary: (BOOL)flag
+{
+  NSTableColumn *tb = [_tableColumns objectAtIndex: column];
+  NSIndexPath *path = [NSIndexPath indexPathForItem: column
+					  inSection: row];
+  NSView *view = [self _renderedViewForPath: path];
+  NSRect drawingRect = [self frameOfCellAtColumn: column
+					     row: row];
+  id item = [self itemAtRow: row];
+	  
+  if (tb == _outlineTableColumn)
+    {
+      drawingRect = [[GSTheme theme] drawOutlineTableColumn: tb
```

Comment:
I don't follow what you mean here.   The reason it passes the drawingRect back out is because if it is the outline view column (i.e. the one that contains the button to open/close the outline) the rect has to be modified to reflect that.

Reply:
Looking at this one again, i see your point about drawing it twice.  Fixing this.

Reply:
It looks as though the code is only rendering the image used to expand the row.   This isn't part of the rowView anyway, from doing tests on macOS.  So we aren't rendering the entire row in the method mentioned here, just that icon... so we should be okay.   I believe that both of these can be resolved... please reopen if you need to.

Reply:
Please ignore the white space corrections.  I did whitespace-cleanup in emacs on the file before committing so it contains some extraneous modifications.